alvr/server_core/src/web_server.rs

```diff
@@ -267,37 +270,7 @@ async fn http_api(
                 .body(latency.to_string().into())?
         }
         "/api/ping" => reply(StatusCode::OK)?,
-        other_uri => {
```

Comment:
why did this exist before? is it for wasm?

Reply:
if it's unused now, i would delete it tbh

Reply:
Ok then

Reply:
Especially since I don't quite trust the fs escape prevention code